src/components/Navbar/__test__/__snapshots__/Navbar.test.js.snap

```diff
@@ -279,11 +279,18 @@ exports[`renders correctly 1`] = `
       <div
         className="flex gap-3 capitalize flex-col md:flex-row"
       >
-        <div
-          className="bg-violet-700 py-2 px-5 text-white rounded-3xl text-sm hover:bg-violet-800 transition-all cursor-pointer"
+        <a
```

Comment:
Use the `<Link` component instead of the anchor tag `<a>`.

Reply:
ok I will change it